doc/source/_toc.yml

```
@@ -100,6 +100,8 @@ parts:
               - file: data/examples/ocr_example
                 title: Scaling OCR with Ray Data
               - file: data/examples/random-access
+              - file: data/examples/batch_inference_object_detection
+                title: Batch Image Object Detection using Ray Data
```

Would be better to clarify that it is a PyTorch example. And don't think we need to explicitly say Ray Data since it's already under the Ray Data section.

```suggestion
                title: Object Detection Batch Inference with PyTorch & torchvision
```

---

Tried this, but looks a bit too length. what about dropping "& torchvision"?